components/search/stories/search.stories.js

```diff
@@ -13,40 +13,56 @@ import { SearchOptions, Template } from "./template.js";
  * This component contains a single input field with both a magnifying glass icon and a clear (“reset”) button displayed within it. When making use of this component, the clear button should only be displayed when the input has a value.
  */
 export default {
-	title: "Search",
-	component: "Search",
+	title: "Search field",
+	description:
+		"This component contains a single input field with both a magnifying glass icon and a \"reset\" button displayed within it.",
+	component: "SearchField",
 	argTypes: {
 		size: size(["s", "m", "l", "xl"]),
-		isQuiet,
 		isDisabled,
-		hasDescription: {
-			name: "Help Text",
+		isHovered,
+		showHelpText: {
+			name: "Show help text",
 			description: "A search field can have help text below the field to give extra context or instruction about what a user should input. The description communicates a hint or helpful information, such as a search’s scope.",
 			type: { name: "boolean" },
 			table: {
 				type: { summary: "boolean" },
-				category: "Advanced",
+				category: "Content",
 			},
 			control: "boolean",
 		},
-		description: {
+		helpTextLabel: {
 			name: "Help text (description)",
 			type: { name: "string" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Content",
+			},
+			control: { type: "text" },
+			if: { arg: "showHelpText", eq: true },
+		},
+		isFocused,
+		isKeyboardFocused,
+		inputValue: {
+			name: "Display value and close button",
+			description: "When defined, this will display the input and the clear button within the search field.",
```

Comment:
```suggestion
			name: "Value",
			description: "When defined, this will set the value of the input, and the clear button will appear within the search field.",
```
I was a little confused by the title of this one at first (partly because it said close button and not clear button). I'd suggest just calling it "Value".

Reply:
I think it also would be helpful to show an example with a value and the clear button on the Docs page.